From 5e925ef55c8be6e20e542fb096e9ded85eb5f67c Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Tue, 14 Apr 2015 19:35:22 +0800
Subject: [PATCH] HDMI: rk_fb_disp_scale is disabled. 	Overscan function is
 achieved by fb driver, so HDMI 	driver do not call this function.

Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 drivers/video/rockchip/hdmi/rockchip-hdmi-lcdc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/video/rockchip/hdmi/rockchip-hdmi-lcdc.c b/drivers/video/rockchip/hdmi/rockchip-hdmi-lcdc.c
index 8a10edb91bb9..571cc31e00e9 100644
--- a/drivers/video/rockchip/hdmi/rockchip-hdmi-lcdc.c
+++ b/drivers/video/rockchip/hdmi/rockchip-hdmi-lcdc.c
@@ -173,11 +173,11 @@ int hdmi_set_lcdc(struct hdmi *hdmi)
 
 	if (rc == 0) {
 		rk_fb_switch_screen(&screen, 1, hdmi->lcdc->id);
-		if (rk_fb_get_display_policy() != DISPLAY_POLICY_BOX)
+/*		if (rk_fb_get_display_policy() != DISPLAY_POLICY_BOX)
 			rk_fb_disp_scale(hdmi->xscale,
 					 hdmi->yscale,
 					 hdmi->lcdc->id);
-	}
+*/	}
 	return rc;
 }
 
-- 
2.35.3

